src/cmd/acme/rows.c

```diff
@@ -22,6 +22,9 @@ static Rune Lcolhdr[] = {
 	0
 };
 
+/* Use unicode PUA to (un)mask ascii */
+#define PMASK(x) (0XF000^x)
```

Comment:
The comment is poor.
/*
 * Use a character from the Unicode Private Use Area to escape newlines
 * within multi-line tags.
 */